### PR TITLE
Add description for CSI pv accessMode

### PR DESCRIPTION
**Pull request by cwdsuzhou**
Add description for CSI pv accessMode

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -343,27 +344,29 @@ In the CLI, the access modes are abbreviated to:
...
+| iSCSI                | &#x2713;     | &#x2713;              | -            |
+| Quobyte              | &#x2713;     | &#x2713;              | &#x2713;     |
+| NFS                  | &#x2713;     | &#x2713;              | &#x2713;     |
+| RBD                  | &#x2713;     | &#x2713;              | -            |
+| VsphereVolume        | &#x2713;     | -                     | - (works when 
+pods are collocated)  |
+| PortworxVolume       | &#x2713;     | -                     | &#x2713;     |
+| ScaleIO              | &#x2713;     | &#x2713;              | -            |
+| StorageOS            | &#x2713;     | -                     | -            |
+|CSI                   | &#x2713;     | depends on the driver |depends on the driver |
```
Comment (Rajakavitha1) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Can we list the Volume Plugin in alphabetical order?

Comment (cwdsuzhou) [reply content/en/docs/concepts/storage/persistent-volumes.md]: thanks,  done

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -343,27 +344,28 @@ In the CLI, the access modes are abbreviated to:
...
-| ScaleIO              | &#x2713;     | &#x2713;    | -            |
-| StorageOS            | &#x2713;     | -           | -            |
+| Volume Plugin        | ReadWriteOnce| ReadOnlyMany          | ReadWriteMany|
+| :---                 |     :---:    |    :---:              |    :---:     |
+| AWSElasticBlockStore | &#x2713;     | -                     | -            |
+| AzureFile            | &#x2713;     | &#x2713;              | &#x2713;     |
+| AzureDisk            | &#x2713;     | -                     | -            |
+| CephFS               | &#x2713;     | &#x2713;              | &#x2713;     |
+| Cinder               | &#x2713;     | -                     | -            |
+| CSI                  | &#x2713;     | depends on the driver | depends on the driver |
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Even rwo depends on the driver

Comment (cwdsuzhou) [reply content/en/docs/concepts/storage/persistent-volumes.md]: Thanks for clarification! updated.

**Pull request merged**